### DISCUSSION TITLE
Very large stochastic error of FC gradient modulus

### DISCUSSION THREAD

**Discussion by kw600** [SSCHAcode/python-sscha#140] [Q&A] [answered]
Dear Sir/Madam,

I am using SSCHA for a few systems, like TiSe2 and high pressure Oxygen. I am always seeing very large stochastic error of FC gradient modulus and it does not decrease in the minimization process. Please see the following output for high presure oxygen c2c phase with 512 configurations:

Number of symmetries before the step:  4
Harmonic contribution to free energy =     493.91843221 meV
Anharmonic contribution to free energy =  -16241.39753186 +-       0.31416460 meV
Free energy =  -15747.47909965 +-       0.31416460 meV
FC gradient modulus =       2.04487461 +-   81634.18839600 bohr^2
Struct gradient modulus =       0.87334436 +-       4.67002584 meV/A
Kong-Liu effective sample size =  505.7889805697602

The minimization seems to become converged but the error here is very large: 'FC gradient modulus =       2.04487461 +-   81634.18839600 bohr^2'.

I am using the default minimizing ways (pre-conditioned CG minimization) and I also tried to use 'root4 + no precondition' but does not really decrease the error. The DFT code I am using is VASP 5.4.4. 

I am wondering what's the issue about the stochastic error and how can I fix it (or it does not matter)?

Many thanks for your help.
Kang

**Comment by mesonepigreco** [comment, marked as answer]
Hi, There is a problem in the estimation of the error of the FC that we are working to fix it and, in some cases, causes the error to be overestimated by order of magnitudes.
It does not affect the minimization (except that the stopping criteria is meet much sooner), and your system seems to be well converged.
To minimize better, reduce the meaningful_factor attribute the SSCHA_Minimizer object so that your gradient can go to zero, but otherwise the minimization is unaffected.

**Reply by kw600** [reply]
That's helpful. Many thanks for your explanation.
